hathor/indexes/memory_deps_index.py

```diff
@@ -34,6 +34,9 @@ class MemoryDepsIndex(DepsIndex):
     _txs_with_deps_ready: set[bytes]
 
     # Next to be downloaded
+    # - Key: hash of the tx to be downloaded
+    # - Value[0]: height
```

Comment:
The base module `deps_index` has a much in-depth explanation, but I think it doesn't explain this, basically it's the height of the block the "requested" that transaction, which was used to prioritize which transactions to download first. I'm almost sure this isn't being used anymore, and it will be removed very soon.